types/react-outside-click-handler/index.d.ts

```diff
@@ -15,7 +15,7 @@ export interface DefaultProps {
 // TODO: Remove partial once DT support TS 3.0, so skipping mandatory listed in default props won't result in compile error
 export interface Props extends Partial<DefaultProps> {
     children: React.ReactNode;
-    onOutsideClick: (e: React.MouseEvent<HTMLElement>) => void;
+    onOutsideClick: (e: MouseEvent) => void;
```

Comment:
I feel like I'm missing something obvious: why don't you need to add `dom` to `lib`? Is it because `es6` includes it?

Reply:
React defines dom interfaces in its global.d.ts, so the types are available (although they're empty). 

> Is it because es6 includes it?

I believe it's the `--target es6` option that would include `dom` lib automatically.

Forgive a possibly dumb question, but this is my first DT contribution and now I feel like I'm missing something obvious, so just for clarification: 
Since definitely typed packages are shipped without tsconfig, adding `dom` to `lib` would be only for the sake of testing with dtslint, right? After the types are installed it is up to the user to make sure they provide correct `lib`/`target` compiler options to support dom lib.

Also, which would be preferable: adding `dom` to `lib` or adding triple-slash directive like `/// <reference lib="dom" />`?

Reply:
1. You are correct, tsconfig doesn't ship with type packages, so users have to provide their own, correct tsconfig.
2. It's still preferable to add `dom` to `lib`. I think.

Reply:
Done